doc/sphinx/language/extensions/compil-steps.rst

```diff
@@ -0,0 +1,103 @@
+.. _compil-steps:
+
+==============================
+How Rocq Processes Input Terms
+==============================
+
+It is in theory possible to write down every Gallina term explicitly
```

Comment:
Do you mean "Gallina" here or "CIC+ term". We usually refer to Gallina as the surface language, but to be honest I'm not sure what the original intent was.